ob_intern/Controller/Short Video Post/table cell/ShortVideoPostTableViewCell.swift

```diff
@@ -18,10 +18,14 @@ class ShortVideoPostTableViewCell: UITableViewCell {
     @IBOutlet weak var labelCaption: UILabel!
 
     let gradient: CAGradientLayer = CAGradientLayer()
+    var profile: Profile!
```

Comment:
```suggestion
    var profile: Profile? 
```
พยายามไม่ force นะครับ เหตุผลคือถ้าค่ามันไม่ถูก initial มีโอกาสทำให้แอพ crash ได้สูงครับ